src/transforms/lua/mod.rs

```diff
@@ -32,7 +32,7 @@ enum V1 {
 pub struct LuaConfigV1 {
     /// Transform API version.
     ///
-    /// Specifying this version ensures that Vector does not break backward compatibility.
+    /// Specifying this version ensures that backward compatibility does not break.
```

Comment:
Language thing that I'm not an expert in but is this better?
```suggestion
    /// Specifying this version ensures that backward compatibility is not broken.
```

Same for the V2 comment.

Reply:
That's better!

Reply:
😅 I always doubt my grasp of the english language